cliquet/__init__.py

```diff
@@ -2,6 +2,7 @@
 """
 import warnings
 import pkg_resources
+import resource
```

Comment:
this bloc of imports groups the *stdlib* imports.

It should be ``from cliquet import resource`` along with others below like utils, etc.